.vscode/extensions.json

```diff
@@ -1,3 +1,3 @@
 {
-  "recommendations": ["eamodio.gitlens", "redhat.vscode-yaml", "trunk.io"]
+  "recommendations": ["redhat.vscode-yaml", "trunk.io"]
```

Comment:
Just sticking to what literally helps using trunk (the former gives us autocomplete in trunk.yaml)